src/schemas.ts

```diff
@@ -4,7 +4,10 @@ export const MenteeApplicationSchema = z.object({
   firstName: z.string().min(1, { message: 'First name cannot be empty' }),
   lastName: z.string().min(1, { message: 'Last name cannot be empty' }),
   email: z.string().email({ message: 'Invalid email address' }),
-  contactNo: z.string().min(1, { message: 'Contact number cannot be empty' }),
+  contactNo: z
+    .string()
+    .min(1, { message: 'Contact number cannot be empty' })
+    .regex(/^\+[1-9]\d{1,14}$/, 'Invalid contact number'),
```

Comment:
@Disura-Randunu Did you check this for multiple countries? Each country has different lengths.

Reply:
Thats why a common length was added. Around 14 digits. Should start with + to indicate the country code

Reply:
We probably need to say that you need to add + then let's remove + validations

Reply:
/^\+[1-9]\d{1,3}\d{6,12}$/
this is a bit more correct than before. can have upto 3 numbers after + which makes the country code. then 6-12 digits which is kind of standard amount of numbers in any country.

Reply:
@anjula-sack updated contactNo validation